### DISCUSSION TITLE
Sonoff mini hang

### DISCUSSION THREAD

**Discussion by Waniaok** [RavenSystem/esp-homekit-devices#1964] [MEPLHAA Scripts Collection]
two pieces of sonoff mini hang for me, each time the same.  cannot be turned on by physical switch, not responding.  in the homekit app they work normally.  I have to reboot them in the HAA Manager app and then moments work normally. Both firmware version 11.8

1:

{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12}]},"1":{"r":[{"g":12,"v":1}]},"b":[{"g":0},{"g":4},{"g":4,"t":0}]}]}

2:
Sonoff MINI with external toggle Switch on GIPO3 (RX) and Contact Sensor connected to GIPO4
{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12}]},"1":{"r":[{"g":12,"v":1}]},"b":[{"g":0},{"g":3},{"g":3,"t":0}]},{"t":5,"f0":[{"g":4,"t":0}],"f1":[{"g":4}]}]}

**Comment by nme77** [comment]
The first 1 works well since years for me.
Fw vers 3.5.12 & 5.0.17.
